src/components/molecules/Settings/Workspace/Plugin/PluginAccordion/index.tsx

```diff
@@ -19,17 +19,17 @@ export type PluginAccordionProps = {
 
 const PluginAccordion: React.FC<PluginAccordionProps> = ({ className, items }) => {
   const theme = useTheme();
-  return (
+  return items ? (
     <Accordion
       className={className}
       allowMultipleExpanded
       itemBgColor={theme.colors.bg[3]}
       items={items?.map(i => {
```

Comment:
Just a personal preference to use clear name of variables (even inside iterations), so it would be:
`items={items?.map(item => {` instead of using `i`
What do you think?

Reply:
I assume others can guess what it is but the clearer the better. I choose your idea. 👍